terratest/run-tests-from-cmdline.md

```diff
@@ -0,0 +1,30 @@
+# Running tests from the command line
+
+The test can also be run from the command line which avoids needing to have a license needed to use the IntelliJ plugin.
+
+## Prerequisites
+- A local clone of this repo
+- Install an appropriate version go (v1.13+)
+- Install dep (```curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh```)
+
+## Setup
+Add {path to repo}/terratest to GOPATH environment variable.
+
+Then:
+- cd to {path to repo}/terratest/src/test
+- run ```dep ensure``` -- this may take a little while and log warnings.
```

Comment:
Single backtick is sufficient for in-line code blocks.  You only need 3 for multi-line things